src/js/GenericPointer.js

```diff
@@ -0,0 +1,32 @@
+/******************************************************************************
+ * Aladin Lite project
+ * 
+ * File GenericPointer.js
+ * 
+ ******************************************************************************/
+
+import { SimbadPointer } from "./SimbadPointer.js";
+import { PlanetaryFeaturesPointer } from "./PlanetaryFeaturesPointer.js";
+
+// allow to call either Simbad or Planetary features Pointers
+export let GenericPointer = (function (view, e) {
+    const xymouse = view.imageCanvas.relMouseCoords(e);
+    let radec = view.wasm.screenToWorld(xymouse.x, xymouse.y);
+    if (radec) {
+        // sky case
+        if (view.aladin.getBaseImageLayer().properties.isPlanetaryBody === false) {
+            const queryRadius = Math.min(1, 15 * view.fov / view.largestDim);
+            console.log('queryRadius "generic pointer": ', queryRadius);
+            SimbadPointer.query(radec[0], radec[1], queryRadius, view.aladin);
+        }
+        // planetary body case
+        else {
+            // TODO: replace with actual value
+            const body = view.aladin.getBaseImageLayer().properties.hipsBody;
+            PlanetaryFeaturesPointer.query(radec[0], radec[1], Math.min(80, view.fov / 20.0), body, view.aladin);
+        }
+    } else {
+            console.log("Cannot unproject at the location you clicked on");
```

Comment:
I do not know if we should log on the console or put a console.warn, console.error or just throw an error ? I would also suggest to add a more explicit/basic message for the user. Maybe 'your click has been discarded as it lies out of the map'.

Reply:
Not mine ^^ 
I also don't get what this means. It appears when the users click outside of the view, right ?

Reply:
That was my code but when I read it I think it is not enough user oriented. That is true, it is when we click out of the view/map. I think the PR is a good reason to also update this info/error message.

Reply:
What about : "The location you clicked on is out of the view." ?

Reply:
It is a good one

Reply:
pushed :)